### ISSUE TITLE
Very long snapshot error message obscures dialog Close button

### ISSUE THREAD

**Issue by ericpromislow** [rancher-sandbox/rancher-desktop#5794]
### Problem Description

I'm experimenting with dropping the `maxLength` field for snapshots from `pages/snapshots/create.vue` and let the backend decide which proposed names are valid and which aren't.

The backend reports that the name is too long, as expected, and the UI displays the message
```
Error: invalid name "<900 alnum characters>"
does not match regex...
```

In this view the `Close` button isn't clickable. The only way I could close the dialog was by bringing up the
dev console and running `document.querySelector('#__layout > div > div > div.dialog-actions.dialog-actions-reverse > button').click()` which our users won't have access to.

### Proposed Solution

Maybe any message that the UI doesn't directly control should be put in a scrollable view.

### Additional Information

This is on the branch `torchiaf:feature/snapshots-description-field` and might not be applicable in an actual release.

**Comment by jandubois** [comment]
This should already be fixed by #5777, which truncates the reported name to the first 30 characters and then adds an ellipsis. It then reports max allowed, and actual length of the name field.

I think this approach should be used in all error messages; we should not include arbitrary amounts of erroneous data in the error message, but rather provide an explanation about what exactly is wrong with it.

**Comment by ericpromislow** [comment]
This is no longer a problem.
